### DISCUSSION TITLE
Using Payload for configuration

### DISCUSSION THREAD

**Discussion by alexblav** [vinodsr/node-red-contrib-tuya-smart-device#116] [Q&A]
Hello,

I'm coming back to you to know if it is possible to provide the "tuya-smart-device" node with a dynamic configuration.

I have several identical entities to which I wish to pass the same order. Is it possible to pass a varible for the ID, key and IP fields?

If so how can I do this?

Thank you very much for your help.

**Comment by vinodsr** [comment]
Try using the generic node . Details are there in the help section in the
node red.

On Fri, 21 Apr 2023 at 1:40 PM, alexblav ***@***.***> wrote:

> Hello,
>
> I'm coming back to you to know if it is possible to provide the
> "tuya-smart-device" node with a dynamic configuration.
>
> I have several identical entities to which I wish to pass the same order.
> Is it possible to pass a varible for the ID, key and IP fields?
>
> If so how can I do this?
>
> Thank you very much for your help.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vinodsr/node-red-contrib-tuya-smart-device/discussions/116>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADQ6OEPNLPPVRRK25O7T6DXCI6FVANCNFSM6AAAAAAXGRIXNY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <vinodsr/node-red-contrib-tuya-smart-device/repo-discussions/116@
> github.com>
>
-- 
Vinod Surendran R
vinodsr.dev
